sources/packages/backend/apps/api/src/route-controllers/institution-user/institution-user.aest.controller.ts

```diff
@@ -61,9 +61,9 @@ export class InstitutionUserAESTController extends BaseController {
    * when available.
    * @returns all filtered institution users.
    */
-  @Get()
+  @Get("institution/:institutionId")
   async getInstitutionUserSummary(
-    @Query("institutionId", ParseIntPipe) institutionId: number,
+    @Param("institutionId", ParseIntPipe) institutionId: number,
```

Comment:
why was it changed from `Query` to `Param`

Reply:
As mentioned in the PR description(3rd checklist point) , the institutionId here is a required value not optional and through the application we have used the required value(s) in Param and optional ones in Query. Hence this one was changed.